index.html

```diff
@@ -3,9 +3,22 @@
 <head>
 	<meta charset="UTF-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
-	<title>My First HTML Page with git</title>
```

Comment:
Could you change title to 'My Best HTML Page with git' ?

Reply:
yes